app/models/partners.rb

```diff
@@ -0,0 +1,5 @@
+module Partners
+  #  def self.table_name_prefix
+  #   "partners_"
+  # end
```

Comment:
Does this module need to exist?

Reply:
(Removes it and runs tests)  Apparently not.